change/@fluentui-react-rating-preview-73090623-b968-461d-857f-78ed02671b60.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30465&commitId=88c981dd7bffc92eb7be50a04e1aa93b7592c2c8&env=prod&clientType=FLUENTUI)


  <details><summary><b>RatingDisplay Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|RatingDisplay Converged.size small with value and count.rest.chromium.png|2245| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>